### DISCUSSION TITLE
How can I prevent ALE from showing error message in comments?

### DISCUSSION THREAD

**Discussion by ruiyan-ma** [w0rp/ale#4429] [Q&A - Ask for help with problems] [answered]
As the following picture shows, I already has a float window to show the error message, but ALE still shows error message in comments (at line 36). How can i stop that? 

<img width="649" alt="image" src="https://user-images.githubusercontent.com/87853503/214719869-1573fc2d-902b-4fb1-a3d3-7426e574bf07.png">

**Comment by hsanson** [comment, marked as answer]
Add this to your configuration:  `let g:ale_virtualtext_cursor = 0`
